setup.py

```diff
@@ -208,12 +208,12 @@ def run(self):
     'appdirs==1.4.0',
     'colorlog==2.7.0',
     'json-rpc==1.10.3',
-    'pycoin==0.62',
+    'pycoin==0.77',
     'pycrypto==2.6.1',
     'pysha3==0.3',
-    'pytest==2.9.1',
+    'pytest==2.9.2',
     'pytest-cov==2.2.1',
-    'python-bitcoinlib==0.5.1',
+    'python-bitcoinlib', #==0.7.0
```

Comment:
I think we should lock this to 0.7.0 so no one accidentally builds counterparty server with an unknown future version of python-bitcoinlib.